ckanext/stcndm/logic/common.py

```diff
@@ -402,10 +434,16 @@ def get_upcoming_releases(context, data_dict):
     start_date = _get_or_bust(data_dict, 'startDate')
     end_date = _get_or_bust(data_dict, 'endDate')
 
-    q = {'q': 'release_date:[{startDate}:00Z TO {endDate}:00Z] AND last_publish_status_code:8'.format(
-        startDate=start_date,
-        endDate=end_date),
-        'rows': 500}
+    q = {
+        'q': (
+            'release_date:[{startDate}:00Z TO {endDate}:00Z] '
+            'AND last_publish_status_code:8'
```

Comment:
(For now until this is all sorted out, with respect to magic business logic explanation)